sidecar/internal/cmd/start.go

```diff
@@ -36,11 +36,11 @@ func init() {
 		"the public port you wish to run the sidecar server on",
 	)
 	startCmd.PersistentFlags().StringVarP(
-		&SsvURLFlag,
-		"ssv-url",
-		"s",
-		"http://127.0.0.1:8888",
-		"the hostname and port of the SSV binary you wish to connect to",
+		&PublicKeyPathFlag,
+		"public-key",
+		"f",
+		"",
+		"the filepath of your SSV node's public key",
```

Comment:
is it the encrypted key just the public key? I think you used "encrypted key" earlier, which I'm not sure how it relates to the public key